src/psyclone/tests/psyir/tools/call_tree_utils_test.py

```diff
@@ -466,10 +466,10 @@ def test_call_tree_utils_const_argument():
 
 # -----------------------------------------------------------------------------
 @pytest.mark.usefixtures("clear_module_manager_instance")
+@pytest.mark.usefixtures("lfric_config")
```

Comment:
You can just add "lfric_config" to the list in the `usefixtures` call above. (I think it's comma-delimited but you'll have to check.)

Reply:
TIL! Done

Reply:
TIL what 'TIL' means :-)